EL-3306-Checkbox-Alignment

ashley-hunter opened the pull request:
Removing outer margin on checkboxes and radio buttons

ashley-hunter commented:
Added utility classes, also noticed toggle switch also had margin bottom which has now also been removed

donna-marijne commented on docs/app/pages/components/components-sections/tree-view/tree-grid/snippets/app.html:
```diff
@@ -9,7 +9,7 @@
                 <tr class="treegrid-header table-header-dark">
                     <th>
-                        <ux-checkbox aria-label="Select all rows" [(value)]="selectAll"></ux-checkbox>
+                        <ux-checkbox class="m-r-sm" aria-label="Select all rows" [(value)]="selectAll"></ux-checkbox>
```
Could you remove the bootstrap class and update the library stylesheet to ensure proper spacing for existing and future implementations of the tree grid?

donna-marijne commented on docs/app/pages/components/components-sections/tree-view/tree-view-custom-node/snippets/app.snippet.html:
```diff
@@ -15,6 +15,7 @@
                 <div class="node-wrapper">
 
                     <ux-checkbox
+                      class="m-r-ms"
```
Similar to the tree grid, please update the stylesheet so that this works out of the box.

donna-marijne commented on docs/app/pages/css/css-sections/forms/inline-form/inline-form.component.html:
```diff
@@ -1,9 +1,9 @@
 <form role="form" class="form-inline" novalidate>
-    <div class="form-group">
+    <div class="form-group m-r-xs">
```
Can this be something like

````less
.form-inline {
  .form-group + .form-group {
    margin-left: 5px;
  }
}
````

ashley-hunter commented:
- Missed bootstrap classes removed.
- Moved tree view/tree grid styling to our stylesheet.
- In regards to the inline form spacing, I have reverted these changes, turns out in JIT mode Angular was removing whitespaces, whereas it wasn't in production mode - I have now made JIT mode retain the whitespaces so this is consistent in both versions - form spacing should now be the same as it used to be.

donna-marijne commented on docs/app/pages/components/components-sections/conduits/multiple-zones/example/inspector/inspector.component.html:
```diff
@@ -4,11 +4,15 @@
+
+        <div class="checkbox-container">
+            <ux-checkbox *ngFor="let zone of zones"
+                class="m-b-xs"
```
I've checked and this can safely be removed.

ashley-hunter replied:
Missed that one, thanks, Fixed now

ashley-hunter commented:
I believe they should now all be addressed (with the exception of the inline form styling which I have explained in the comment above)

donna-marijne commented:
Tree grid and tree view are still using bootstrap classes to compensate for right margin alterations.

ashley-hunter commented:
Perhaps I'm looking in the wrong place, but from what I can see they have been removed from all the checkboxes?

https://github.com/UXAspects/UXAspects/blob/EL-3306-Checkbox-Alignment/docs/app/pages/components/components-sections/tree-view/tree-grid/tree-grid.component.html
![image](https://user-images.githubusercontent.com/20795331/49581382-b20e2380-f949-11e8-8d00-5c989b6a0063.png)
![image](https://user-images.githubusercontent.com/20795331/49581636-82abe680-f94a-11e8-8c85-35441c41c817.png)


https://github.com/UXAspects/UXAspects/blob/EL-3306-Checkbox-Alignment/docs/app/pages/components/components-sections/tree-view/tree-view-custom-node/tree-view-custom-node.component.html
![image](https://user-images.githubusercontent.com/20795331/49581410-cce09800-f949-11e8-86c9-514a2e1951ba.png)
